src/lib/features/dependent-features/dependent-features-service.ts

```diff
@@ -17,20 +17,20 @@ export class DependentFeaturesService {
     }
 
     async upsertFeatureDependency(
-        parentFeature: string,
+        childFeature: string,
```

Comment:
there was a mistake in the previous version. child should have a dependency on parent (dependent feature)